### DISCUSSION TITLE
What is in Telex, in option menu... how do you turn it on?

### DISCUSSION THREAD

**Discussion by Noboru00** [flybywiresim/aircraft#4925] [Help] [answered]
I have no aircraft in efb moving map. Flybywire v0.6.1.

Thank You
Noboru

**Comment by hotshotp** [comment, marked as answer]
this is an issue with msfs refresh rates as far as i know. i thought you meant on the fbw website map thats why i said telex before, but if you mean just the plane icon in the efb then its a known issue as of now

**Reply by Noboru00** [reply]
Roger that... the efb knows where I'm at far as an airport , but no plane displayed.

Thank You My Friend.
Noboru
